### ISSUE TITLE
gem update failure on OSX

### ISSUE THREAD

**Issue by djberg96** [rubygems/rubygems#1762]
I'm not having any luck updating from 2.6.4 to 2.6.7 with Ruby 2.3.1 OSX (10.12.1). I am using rbenv (environment info below).

```
>gem -v
2.6.4

>gem update --system
Updating rubygems-update
Fetching: rubygems-update-2.6.7.gem (100%)
Successfully installed rubygems-update-2.6.7
Parsing documentation for rubygems-update-2.6.7
Installing ri documentation for rubygems-update-2.6.7
Installing darkfish documentation for rubygems-update-2.6.7
Done installing documentation for rubygems-update after 22 seconds
Parsing documentation for rubygems-update-2.6.7
Done installing documentation for rubygems-update after 0 seconds
Installing RubyGems 2.6.4
RubyGems 2.6.4 installed
Parsing documentation for rubygems-2.6.4
Installing ri documentation for rubygems-2.6.4

>gem -v
2.6.4 # wut?
```

```
>gem env
RubyGems Environment:
  - RUBYGEMS VERSION: 2.6.4
  - RUBY VERSION: 2.3.1 (2016-04-26 patchlevel 112) [x86_64-darwin15]
  - INSTALLATION DIRECTORY: /Users/dberger/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0
  - USER INSTALLATION DIRECTORY: /Users/dberger/.gem/ruby/2.3.0
  - RUBY EXECUTABLE: /Users/dberger/.rbenv/versions/2.3.1/bin/ruby
  - EXECUTABLE DIRECTORY: /Users/dberger/.rbenv/versions/2.3.1/bin
  - SPEC CACHE DIRECTORY: /Users/dberger/.gem/specs
  - SYSTEM CONFIGURATION DIRECTORY: /Users/dberger/.rbenv/versions/2.3.1/etc
  - RUBYGEMS PLATFORMS:
    - ruby
    - x86_64-darwin-15
  - GEM PATHS:
     - /Users/dberger/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0
     - /Users/dberger/.gem/ruby/2.3.0
  - GEM CONFIGURATION:
     - :update_sources => true
     - :verbose => true
     - :backtrace => false
     - :bulk_threshold => 1000
     - "install" => "--no-document"
     - "backtrace" => true
     - "benchmark" => false
     - "verbose" => true
     - "update_sources" => true
     - "sources" => ["http://rubygems.org"]
  - REMOTE SOURCES:
     - https://rubygems.org/
  - SHELL PATH:
     - /Users/dberger/.rbenv/versions/2.3.1/bin
     - /Users/dberger/.rbenv/libexec
     - /Users/dberger/.rbenv/plugins/ruby-build/bin
     - /Users/dberger/.rbenv/plugins/rbenv-gemset/bin
     - /Users/dberger/.rbenv/shims
     - /Users/dberger/.rbenv/bin
     - /usr/local/opt/ruby/bin
     - /usr/local/bin
     - /usr/bin
     - /bin
     - /usr/sbin
     - /sbin
     - /opt/X11/bin
     - /Library/TeX/texbin
```

**Comment by segiddins** [comment]
This was a bug fixed in 2.6.7, as a one-time fix run `gem update --system=2.6.7`

**Comment by djberg96** [comment]
@segiddins That worked, thanks.
